content/en/kubernetes.md

```diff
@@ -5,9 +5,9 @@ category: technology
 ---
 
 ## What it is
-Kubernetes, often abbreviated as K8s, is a popular open-source tool for modern infrastructure automation. It's like a data center operating system that manages applications running across a [distributed system](/distributed_systems/) (just like the OS on your laptop that manages your apps). 
+Kubernetes, often abbreviated as K8s, is a container orchestrator which is an open-source tool that automates the life-cycle of [container](/container/) applications on modern infrastructures. It's like a data center operating system that manages applications running across a [distributed system](/distributed_systems/) (just like the OS on your laptop that manages your apps). 
```

Comment:
```suggestion
Kubernetes, often abbreviated as K8s, is an open source container orchestrator that automates the life-cycle of [containerized](/container/) applications on modern infrastructures. It's like a data center operating system that manages applications running across a [distributed system](/distributed_systems/) (just like the OS on your laptop that manages your apps). 
```

Reply:
thanks, @CathPag that sounds much better